canonical/ubuntu.com
# Meganav bug fixes

## Done

- Fixed a number of small bugs on the new meganav, including:
  - Fixed broken layout when transitioning between small and large breakpoints
  - Fixed keyboard navigation being impossible
  - Fixed opening "All Canonical" menu failing to close other open meganav sections

## QA

- Visit https://ubuntu-com-12390.demos.haus/
- Test the meganav at various breakpoints, the layout shouldn't break
- Test that you can navigate successfully via both mouse and keyboard
- Visit /openstack
- See that the meganav still works well in the "reduced" navigation view

## Issue / Card

Fixes https://github.com/canonical/web-squad/issues/6178
